plugins/idea/src/main/java/com/oracle/plugin/jtreg/components/JTRegFileManagerListener.java

```diff
@@ -134,6 +140,15 @@ void processFileOpened(TestInfo testInfo) {
                         rootModel.addSourceFolders(testRoots);
                     }
                 }
+                if (isTestNg || isJUnit) {
+                    String dir = "file://" + JTRegService.getInstance(project).getJTRegDir() + File.separator + "lib";
+                    Library library = JTRegUtils.createJTRegLibrary(project, dir, dir);
+                    testInfo.jtregLib = library;
+                    rootModel.addLibrary(library);
+                } else if (testInfo.jtregLib != null) {
+                    rootModel.removeLibrary(testInfo.jtregLib);
```

Comment:
I'm not sure this is logically correct. While I can understand adding jtreg/testng library to the model as soon as a new test file requiring it is opened, removing the library seems more problematic. In principle we should remove the testng/junit library from the model only if there's no open test that's a jtreg test. Which doesn't seem to be what's happening in the code.